### PR TITLE
CRIMAPP-757 Send dependant question to the datastore

### PR DESCRIPTION
**Pull request by timpeat**
## Description of change

Use stored value for "client_has_dependants" and report to datastore.

## Link to relevant ticket

[CRIMAPP-757](https://dsdmoj.atlassian.net/browse/CRIMAPP-757)

## Notes for reviewer

Part of https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/585

## How to manually test the feature

[CRIMAPP-757]: https://dsdmoj.atlassian.net/browse/CRIMAPP-757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/services/adapters/structs/applicant.rb [hunk 1]
```diff
@@ -12,7 +12,8 @@ def correspondence_address
       def serializable_hash(options = {})
         super(
           options.merge(
-            methods: []
+            methods: [],
+            except: [:relationship_to_someone_else, :residence_type]
```
Comment (arthurashman) [inline app/services/adapters/structs/applicant.rb:16]: Is this meant to be in this PR?

File: spec/services/adapters/structs/income_details_spec.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 RSpec.describe Adapters::Structs::IncomeDetails do
   subject { application_struct.income }
 
-  let(:application_struct) { build_struct_application }
+  let(:application_struct) { build_struct_application(with_full_means: false) }
```
Comment (arthurashman) [inline spec/services/adapters/structs/income_details_spec.rb:6]: Do we have a test in here that also checks when we do have full means?

File: app/services/adapters/structs/applicant.rb (hunk 1 above)
Comment (hiboabd) [reply app/services/adapters/structs/applicant.rb:16]: I've just merged my pr so you should be able to get rid of this

File: spec/services/adapters/structs/income_details_spec.rb (hunk 2 above)
Comment (timpeat) [reply spec/services/adapters/structs/income_details_spec.rb]: Well spotted. I disabled it when debugging. Have sorted now. Thank you :)

**Review by arthurashman — APPROVED** [review]

**Pull request merged**